Dockerfile

```diff
@@ -12,7 +12,8 @@ RUN wget -nv http://max-assets.s3-api.us-geo.objectstorage.softlayer.net/deeplab
 
 RUN pip install numpy && \
     pip install tensorflow && \
-    pip install pillow
+    pip install pillow && \
+    pip install flask-cors
```

Comment:
If this package is needed for all the models we should put it in the base image. Maybe you could open a PR here? https://github.com/codait/max-base

Reply:
...also what is pillow installed for?

Reply:
I didn't add pillow (just the slash on that line), but I believe its a Tensorflow dep?

Reply:
I'll take a look at `codait/max-base` and PR there if applicable 👍

Reply:
My mistake, I misread it and thought you had added it.

Reply:
Are you going to remove this since you have the PR open for the base image?